src/fides/api/ops/util/data_category.py

```diff
@@ -1,9 +1,38 @@
 from enum import Enum as EnumType
-from typing import Type
+from typing import List, Type
 
 from fideslang import DEFAULT_TAXONOMY
+from fideslang.models import DataCategory as FideslangDataCategory
+from sqlalchemy.orm import Session
 
+from fides.api.ctl.sql_models import DataCategory as DataCategoryDbModel  # type: ignore
 from fides.api.ops import common_exceptions
+from fides.core.config import get_config
+from fides.lib.db.session import get_db_session
+
+
+def get_fides_data_categories_from_db() -> List[FideslangDataCategory]:
+    """
+    Returns each DataCategory from the database as it would have been defined in the
+    default taxonomy.
+    """
+    db: Session = get_db_session(get_config())()
+    all_categories = DataCategoryDbModel.all(db)
+    db.close()
```

Comment:
I don't think we should be creating a new session here, this also creates a new engine just to get the data categories.  The rule target endpoint has the session injected - I think we should pass that session to a method that validates the data category instead.